GameOfRunes/GameOfRunes/Custom Nodes/Player Area/PlayerAreaNode.swift

```diff
@@ -70,19 +92,40 @@ class PlayerAreaNode: SKSpriteNode {
             layoutManaBar()
         }
     }
+    var powerUpContainerPositionOffsetFromCenter: CGVector {
+        didSet {
+            layoutPowerUpContainer()
+        }
+    }
+    var summonNodePositionOffsetFromCenter: CGVector {
+        didSet {
+            layoutSummonNode()
+        }
+    }
     
     init(size: CGSize = .zero, position: CGPoint = .zero) {
         healthBarNode = .init()
         manaBarNode = .init()
-        healthBarSize = size.applying(.init(scaleX: 0.45, y: 0.4))
-        manaBarSize = size.applying(.init(scaleX: 0.45, y: 0.4))
-        healthBarPositionOffsetFromCenter = CGVector(dx: -size.width / 4.5, dy: size.height / 4.5)
-        manaBarPositionOffsetFromCenter = CGVector(dx: size.width / 4.5, dy: size.height / 4.5)
+        powerUpContainerNode = .init(powerUpTypes: [.darkVortex, .hellfire, .icePrison])
+        summonNode = .init()
+        
+        healthBarSize = size.applying(.init(scaleX: 0.45, y: 0.325))
+        manaBarSize = size.applying(.init(scaleX: 0.45, y: 0.325))
+        powerUpContainerSize = size.applying(.init(scaleX: 0.45, y: 0.5))
+        summonNodeSize = summonNode.size.scaleTo(height: size.height * 0.5)
+        
+        healthBarPositionOffsetFromCenter = .init(dx: -size.width / 4.5, dy: size.height / 4)
+        manaBarPositionOffsetFromCenter = .init(dx: size.width / 4.5, dy: size.height / 4)
+        powerUpContainerPositionOffsetFromCenter = .init(dx: -size.width / 4.5, dy: -size.height / 5.5)
+        summonNodePositionOffsetFromCenter = .init(dx: size.width / 4.5, dy: -size.height / 5.5)
```

Comment:
Maybe the 4.5 and 5.5 can be shifted into GameConfig?

Reply:
Hmm yes, will do it in the next PR